types/camelcase-keys-deep/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for camelcase-keys-deep 0.1
+// Project: https://github.com/rxaviers/camelcase-keys-deep
+// Definitions by: Søren Englund <https://github.com/englund0110>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+declare function camelcaseKeysDeep(obj: object): {};
+
+declare namespace camelcaseKeysDeep {}
```

Comment:
this should be removed once you switch the import syntax in the tests. It's a hack that we can't remove from Typescript because of backward compatibility.

Reply:
I have removed it. Thanks!